### DISCUSSION TITLE
Private Assistant settings not fully works

### DISCUSSION THREAD

**Discussion by Astricaelus** [danny-avila/LibreChat#2918] [Troubleshooting] [answered]
### What happened?

I have just upgraded from previous version to latest version and set privateAssistants to true according to <https://github.com/danny-avila/LibreChat/pull/2831/files#diff-cf4f55b3b2cc60c5b70b3bbe4dabf412a7922213c13bf23d12205e6a833fe9e7>. This works for new users, that is, new users cannot view assistants created by other users, either new or old. But it seems that old users can still see assistants created by other users.

### Steps to Reproduce

1. Directly upgrade from older versions and keeps the data.
2. create a new user
3. The old user create an assistant, and the new user cannot see it. That correct.
4. The new user create an assistant, and the old user can see the assistant.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I can't reproduce it. Your "old" users, do they have the "ADMIN" role in the database? This is the only way they could view private assistants, new or old.

**Reply by Astricaelus** [reply]
Thanks! Is the first created user have "ADMIN" role in the database? Or how can I directly view the role of users in the database?

**Reply by danny-avila** [reply, marked as answer]
Yes, the first created user is given the ADMIN role. This will play a more prominent, obvious role in the future. You would have to view the "users" collection in the database. If you are using mongodb atlas, you can do this through their site. If using the database locally, there are many ways to do this. 

One of the recommended ways to do this is to use Mongo Express:
https://www.librechat.ai/blog/2023-11-30_mongoexpress
